Sensors.cpp

```diff
@@ -0,0 +1,47 @@
+#include <iostream>
+#include <string.h>
+#include "Sensors.h"
+#include "SensorConstants.h"
+
+bool SensorBase::initialize(){
+    if(sensorID < MAXSAMPLEFILES){
+        sensorInputs.open(readings[sensorID], std::ifstream::in);
```

Comment:
For me take some time to understand the relation over data between SID and readings array